docker-compose/owls/.env

```diff
@@ -1,10 +1,11 @@
-# Image tags
 COMPOSE_PROJECT_NAME=owls
+
+# Image tags
 OWSEC_TAG=main
 OWLS_TAG=main
 OWLSUI_TAG=master
 KAFKA_TAG=2.8.0-debian-10-r43
-ZOOKEEPER_TAG=latest
+ZOOKEEPER_TAG=3.8
```

Comment:
Are we still having Zookeper deployed?

Reply:
Yes :-( I'm changing it to use postgres by default. Maybe the change to kraft should be next in line then?